internal/util/ssa/managedfields_test.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package ssa contains utils related to ssa.
+package ssa
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestCleanUpManagedFieldsForSSAAdoption(t *testing.T) {
+	ctx := context.Background()
+
+	ssaManager := "test-manager"
+	classicManager := "manager"
```

Comment:
```suggestion
	ssaManager := "ssa-manager"
	classicManager := "classic-manager"
```
\+ update names below please so that we are only talking about ssa and classic manager and not also about "test-manager"

Reply:
`classicManager` has to remain `manager` because that is what the function is designed to look for. I am open to suggestions to rename the variable `classicManager` to something better if you have any idea.

Reply:
Ah good point. That's fine. Let's just get rid of test manager